### PR TITLE
Add iOS tintColor prop to TextInput

### PR DESCRIPTION
**Pull request by jordanranz**
In response to https://github.com/facebook/react-native/issues/5595

Adds tintColor property to TextInput component for iOS:

screen shot 2016-01-28 at 1 39 35 pm

Usage:

screen shot 2016-01-28 at 1 42 37 pm

### CODE REVIEW

File: Libraries/Text/RCTTextFieldManager.m [hunk 1]
```diff
@@ -78,6 +78,7 @@ - (BOOL)textFieldShouldEndEditing:(RCTTextField *)textField
 RCT_REMAP_VIEW_PROPERTY(editable, enabled, BOOL)
 RCT_EXPORT_VIEW_PROPERTY(placeholder, NSString)
 RCT_EXPORT_VIEW_PROPERTY(placeholderTextColor, UIColor)
+RCT_EXPORT_VIEW_PROPERTY(tintColor, UIColor)
```
Comment (nicklockwood) [inline Libraries/Text/RCTTextFieldManager.m]: Can you expose this in RCTTextViewManager.m too please, as TextInput.js wraps them both.

**Comment by nicklockwood** [comment]
Could you add an example to TextInputExample.js demoing the feature? Thanks.

File: Libraries/Text/RCTTextFieldManager.m (hunk 1 above)
Comment (jordanranz) [inline Libraries/Text/RCTTextFieldManager.m]: Yep, good call.

**Comment by nicklockwood** [comment]
Thanks!

@facebook-github-bot shipit

**Comment by dmmiller** [comment]
Hi @jordanranz Please note that I updated this to selectionColor and made it work on Android in https://github.com/facebook/react-native/commit/0c91931adf15fb456e72d08dca872f632661b655
Thanks for the iOS bit which kicked us into gear to do Android as well.

**Pull request closed without merging**